gateway/modules/crud/operations.go

```diff
@@ -54,7 +58,21 @@ func (m *Module) Read(ctx context.Context, dbAlias, project, col string, req *mo
 		return nil, err
 	}
 
-	n, result, err := crud.Read(ctx, project, col, req)
+	var n int64
+	var result interface{}
+	var dataLoader *dataloader.Loader
+	var key model.ReadRequestKey
+	if req.Options != nil {
+		if req.Options.Prefix != "" {
+			key = model.ReadRequestKey{DBType: dbAlias, Col: col, HasOptions: req.Options.HasOptions, Req: *req}
+			dataLoader = newLoaderMap().get(fmt.Sprintf("%s-%s", req.Options.Prefix, uuid.New().String()), m)
+			result, err = dataLoader.Load(ctx, key)()
+			// clear the key
+			_ = dataLoader.Clear(ctx, key)
```

Comment:
You shouldn't clear the key here. The user of the crud module will call clear this key.